Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM blackducksoftware/hub-docker-common:1.0.1 as docker-common
+FROM blackducksoftware/hub-docker-common:1.0.3 as docker-common
```

Comment:
I am assuming a subsequent PR on hub-backend will bump the versions?

Reply:
Of this container, yes